src/test/java/com/philips/research/bombar/core/domain/licenses/ObligationsAnalyzerTest.java

```diff
@@ -0,0 +1,123 @@
+package com.philips.research.bombar.core.domain.licenses;
+
+import com.philips.research.bombar.core.domain.Dependency;
+import com.philips.research.bombar.core.domain.Project;
+import com.philips.research.bombar.core.domain.Relation;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ObligationsAnalyzerTest {
+    private static final String OTHER_OBLIGATION = "Other";
+    private static final String OTHER_OBLIGATION_DESC = "Other desc";
+    private static final String KEY = "Key";
+    private static final String TITLE = "Title";
+    private static final String LICENSE = "License";
+    private static final String OBLIGATION = "Obligation";
+    private static final String OBLIGATION_DESC = "Obligation desc";
```

Comment:
Could refactor this into a constant `DESCRIPTION = "Description"`, as the "description" aspect of this constant is more important than the "obligation" aspect.